chirp_data.py

```diff
@@ -0,0 +1,91 @@
+import pickle
+from chirps import *
+
+class ChirpData:
```

Comment:
Food for thought. Will you ever need to create more than one instance of the `ChirpData` class?  If not, consider making this a module instead.